scripts/CMakeLists.txt

```diff
@@ -27,57 +35,81 @@ if (UPDATE_DEPS)
         endif()
         set(_target_arch x64)
     endif()
+    list(APPEND update_dep_command "--arch")
+    list(APPEND update_dep_command "${_target_arch}")
 
     if (NOT CMAKE_BUILD_TYPE)
         message(WARNING "CMAKE_BUILD_TYPE not set. Using Debug for dependency build type")
         set(_build_type Debug)
     else()
         set(_build_type ${CMAKE_BUILD_TYPE})
     endif()
+    list(APPEND update_dep_command "--config")
+    list(APPEND update_dep_command "${_build_type}")
+
+    set(UPDATE_DEPS_DIR_SUFFIX "${_build_type}")
+    if (ANDROID)
+        set(UPDATE_DEPS_DIR_SUFFIX "android/${UPDATE_DEPS_DIR_SUFFIX}")
+    endif()
+    set(UPDATE_DEPS_DIR "${PROJECT_SOURCE_DIR}/external/${UPDATE_DEPS_DIR_SUFFIX}" CACHE PATH "Location where update_deps.py installs packages")
+    list(APPEND update_dep_command "--dir" )
+    list(APPEND update_dep_command "${UPDATE_DEPS_DIR}")
 
-    set(_build_tests_arg "")
     if (NOT BUILD_TESTS)
-        set(_build_tests_arg "--optional=tests")
+        list(APPEND update_dep_command "--optional=tests")
     endif()
 
-    message("********************************************************************************")
-    message("* NOTE: Adding target vl_update_deps to run as needed for updating             *")
-    message("*       dependencies.                                                          *")
-    message("********************************************************************************")
+    if (UPDATE_DEPS_SKIP_EXISTING_INSTALL)
+        list(APPEND update_dep_command "--skip-existing-install")
+    endif()
 
-    set(update_dep_py "${CMAKE_CURRENT_LIST_DIR}/update_deps.py")
-    set(known_good_json "${CMAKE_CURRENT_LIST_DIR}/known_good.json")
+    list(APPEND cmake_vars "CMAKE_TOOLCHAIN_FILE")
+    if (ANDROID)
+        list(APPEND cmake_vars "ANDROID_PLATFORM" "CMAKE_ANDROID_ARCH_ABI" "CMAKE_ANDROID_STL_TYPE" "CMAKE_ANDROID_RTTI" "CMAKE_ANDROID_EXCEPTIONS" "ANDROID_USE_LEGACY_TOOLCHAIN_FILE")
+    endif()
```

Comment:
Not necessary - android isn't supported in this repo. Its fine to keep it though so things are consistent across code bases.